frontend/src/ui/player/Paella.tsx

```diff
@@ -34,7 +34,7 @@ const PaellaPlayer: React.FC<PaellaPlayerProps> = ({ event }) => {
     useEffect(() => {
         // If the ref is not set yet (which should not usually happen), we do
         // nothing.
-        if (!ref.current) {
+        if (!ref.current || !event.authorizedData) {
```

Comment:
That shouldn't be necessary. `<Paella />` should only be usable with an fully authorized event. The prop type needs to be adjusted to make `authorizedData` non-optional. You can probably use TS helpers like `Pick`, `Omit`, `Required`, and stuff like that.

Reply:
Erm, yeah. I agree that it should be possible with a combination of these helpers. But: that turned out to be a little more complicated and would need multiple changes in other types and or props. So I just wrote out the `authorizedData` prop for the `PlayerEvent` again, hope that's ok.